receiver/saphanareceiver/README.md

```diff
@@ -1,10 +1,14 @@
 # SAP HANA Receiver
 
-| Status                   |               |
-| ------------------------ |---------------|
-| Stability                | [development] |
-| Supported pipeline types | metrics       |
-| Distributions            |               |
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
```

Comment:
@dehaansa, this would be a followup PR, but do you think this ought to be bumped to alpha?